addon/components/file-renderer/template.hbs

```diff
@@ -0,0 +1,2 @@
+<iframe src={{mfrUrl}} width={{width}} height={{height}} scrolling="no" marginheight="0" frameborder="0">
```

Comment:
@pattisdr might be a good idea to provide reasonable defaults in the component.js,
i.e. 
```
height: '100%',
width: '100%'
```